src/mame/drivers/pcw.cpp

```diff
@@ -246,7 +246,7 @@ void pcw_state::pcw_update_read_memory_block(int block, int bank)
 	address_space &space = m_maincpu->space(AS_PROGRAM);
 	char block_name[10];
 
-	sprintf(block_name,"bank%d",block+1);
+	snprintf(block_name, sizeof(block_name), "bank%d", block + 1);
```

Comment:
Eugh, isn't this a perfect case for `required_memory_bank_array` so you get benefits in validation, performance, and expressiveness?

Reply:
You already did, didn't you :)